plugins/lookup/conjur_variable.py

```diff
@@ -277,11 +296,16 @@ def run(self, terms, variables=None, **kwargs):
             with open(conf['authn_token_file'], 'rb') as f:
                 token = f.read()
 
-        return _fetch_conjur_variable(
+        conjur_variable = _fetch_conjur_variable(
             terms[0],
             token,
             conf['appliance_url'],
             conf['account'],
             validate_certs,
             cert_file
         )
+
+        if file_mode:
+            return _store_in_file(conjur_variable)
+        else:
```

Comment:
`else` isn't needed at this point since there's only a single logic path that remain after file_mode return

Reply:
Thanks, removed.